docs/GuideTVOS.md

```diff
@@ -0,0 +1,89 @@
+---
+id: guide-tvos
+title: Apple TV changes
```

Comment:
Is the intention of the document to cover what changes need to be done to iOS code in order to support Apple TV? Maybe we can use a better title here, what about "Targeting Apple TV" or "Building Apple TV apps". The filename should also generally match whatever title we use here. Same for the id.